Storage/CertStore.swift

```diff
@@ -0,0 +1,31 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import Foundation
+import Shared
+import UIKit
+import Deferred
+
+/// In-memory certificate store.
+public class CertStore {
+    private var keys = Set<String>()
+
+    public init() {}
```

Comment:
Looks like we do (I think because it's in a different target): ` 'CertStore' cannot be constructed because it has no accessible initializers`